api/email.py

```diff
@@ -33,11 +34,11 @@ def post(self, request):
         Creates a new feedback email and sends it to admins
         """
         data = request.data
-        required = ['message',]
+        required = ['message', ]
```

Comment:
It doesn't make a difference either way.  This particular placement of a comma is sorta a religion issue.  All of the computer languages that I can think of support this construct.  Personally, I don't like having commas at the end of my lists, however this codebase is replete with examples like this.